docs/reference/module-api/module-overview.md

```diff
@@ -257,14 +257,231 @@ modules.export = {
 };
 ```
 
-## Customization functions
+## Initialization function
 
 ### `async init(self)`
-### `beforeSuperClass(self)`
+
+This function runs once when the Apostrophe app first start up. It takes the module, as `self`, as an argument.
+
+While [customization functions](#customization-functions) add functionality for the module in specific ways, `init` provides a space for more open code execution. It is useful for setting properties on the module that could not be set in other sections.
+
+```javascript
+// modules/product/index.js
+module.exports = {
+  // ...
+  options: {
+    alias: 'product',
+    themeColor: 'blue'
+  },
+  init(self) {
+    // 👇 Making `self.options.themeColor` available on `apos.product.theme`.
+    self.theme = self.options.themeColor;
+
+    // 👇 Adding a data migration related to this module using a method from a
+    // separate module.
+    self.apos.migration.add('blurb', async () => {
+      await self.apos.migration.eachDoc({
+        type: 'product'
+      }, async (doc) => {
+        if ((doc.category) === 'old-category') {
+          doc.category = 'new-category';
```

Comment:
Setting doc.category in this way has no effect on the database, only what is in updateOne matters, so this should become part of the `$set` object.

Reply:
Yep. Lazy code copying.